### PR TITLE
Cleanup frameworks

### PR DESCRIPTION
**Pull request by ralphtheninja**
Resolves parts of #9

### CODE REVIEW

File: lib/control-app.js [hunk 1]
```diff
@@ -74,10 +74,6 @@ module.exports = function (config, cb) {
   // airtap files
   app.use('/__zuul', express.static(__dirname + '/../frameworks'))
 
-  // framework files
-  var frameworkDir = path.resolve(__dirname, '../frameworks/tape')
-  app.use('/__zuul', express.static(frameworkDir))
```
Comment (ralphtheninja) [inline lib/control-app.js]: This `app.use()` is no longer needed since we no longer serve `tape/framework.js`

**Comment by ralphtheninja** [comment]
LOADS of red! :man_dancing:

File: lib/control-app.js [hunk 2]
```diff
@@ -1,139 +1,135 @@
...
+    // coverage endpoint
+    app.use('/__zuul/coverage', im.createHandler())
+  }
 
-    var clientBundler = browserify(opt);
-    clientBundler.require(path.join(framework_dir, '/client.js'), { entry: true });
+  var map = undefined
+  var clientBundler = browserify(opt)
+  var tape = path.resolve(__dirname, '../frameworks/tape.js')
+  clientBundler.require(tape, { entry: true })
```
Comment (vweevers) [inline lib/control-app.js]: Do we still need the `.require`, which makes tape available from outside the bundle? Can perhaps simplify it to:

```js
var tape = path.resolve(__dirname, '../frameworks/tape.js')
var clientBundler = browserify(tape, opt)
```

Comment (ralphtheninja) [reply lib/control-app.js]: Works well!

Comment (ralphtheninja) [reply lib/control-app.js]: Btw, if we are to remove the user interface we should just cache this bundle in memory and not bundle at request.

Comment (ralphtheninja) [reply lib/control-app.js]: Then we could also get rid of `watchify`

Comment (vweevers) [reply lib/control-app.js]: There's a use case for watchify though? Not having to restart the server during development.

File: lib/control-app.js [hunk 3]
```diff
@@ -83,9 +83,8 @@ module.exports = function (config, cb) {
   }
 
   var map = undefined
-  var clientBundler = browserify(opt)
   var tape = path.resolve(__dirname, '../frameworks/tape.js')
-  clientBundler.require(tape, { entry: true })
+  var clientBundler = browserify(tape, opt)
```
Comment (vweevers) [inline lib/control-app.js]: 👍

File: lib/control-app.js (hunk 2 above)
Comment (ralphtheninja) [reply lib/control-app.js]: Yep. I was thinking that if you are testing on many browsers (not in development) you want this to be fast. Just thinking out loud.

Comment (vweevers) [reply lib/control-app.js]: That's true. So that's another case for a CI-only option (like `loopback`).

**Pull request merged**